documentation/Experian Commerce Cloud Integration Manual.md

```diff
@@ -368,4 +378,4 @@ Checkout Form | ✔ | ✔ | ✔ | ✔
 * Timeout connection issues can be experience due to a slow internet connection. If you don’t get an answer in the default time; you can try making the request again.
 
 ##  7\. Failover and recovery
-* If the Experian data quality libraries connection fails the store will continue with it's normal workflow; this issue may be caused due a non valid token or slow internet connection.
+* If the unicorn/pegasus libraries connection fails the store will continue with it's normal workflow; this issue may be caused due a non valid token or slow internet connection.
```

Comment:
Let's keep this as Experian Data Quality libraries, since customers won't have any idea of what we mean by Unicorn and Pegasus.